fastlane/lanes/release.rb

```diff
@@ -107,7 +107,12 @@
     attempts = 0
     begin
       attempts += 1
-      set_branch_protection(repository: GITHUB_REPO, branch: release_branch_name)
+      copy_branch_protection(
+        repository: GITHUB_REPO,
+        from_branch: DEFAULT_BRANCH,
+        to_branch: release_branch_name,
+        github_token: get_required_env('GITHUB_TOKEN')
```

Comment:
`github_token` could have been omitted given the action falls back to the the same env var. I decided to explicitly set it to make the whole thing clearer. Otherwise, one might have asked how could the action make changes on GitHub.

Reply:
I like the explicitness too 👍 

Another approach we have been using in Tumblr is to [start our release-related lanes with `require_env_vars(<the list of env vars this lane relies on>)`](https://github.tumblr.net/Tumblr/ios/blob/develop/fastlane/Fastfile#L95). This is a very simple [local helper](https://github.tumblr.net/Tumblr/ios/blob/develop/fastlane/lib/helpers.rb#L12-L20) but helped anyone who was covering for Tumblr Releases and new to the process to ensure they had the required env vars configured in their `~/.tumblr-env.default` before starting the lane (as opposed to risking have the lane start running its first tasks but then fail due to missing value in the middle of the lane, having to undo the partial changes manually before retrying…)

With developer-led releases and the lanes being run by CI and not from people's own computers anymore this `require_env_vars` preflight check approach will probably become less necessary/useful though, while just providing the parameter explicitly like you did here would be enough.